src/compose/ComposeBox.js

```diff
@@ -128,7 +129,10 @@ class ComposeBox extends PureComponent {
         <AutoCompleteView text={message} onAutocomplete={this.handleAutoComplete} />
         <View style={[styles.composeBox, { height: totalHeight }]}>
           <View style={componentStyles.bottom}>
-            <ComposeMenu />
+            <ComposeMenu
+              handleMessageChange={this.handleMessageChange}
```

Comment:
You mean I should wire up the redux in ComposeMenu, as atm it's not possible to call actions from ComposeMenu?